src/account_multisig.ts

```diff
@@ -150,7 +217,27 @@ export class AccountMultisig extends Account {
         return requestId
     }
 
-    async verifyRequestCode(securityCode: string) {
+    async getCodeDefault(method: any): Promise<string> {
+        throw new Error('There is no getCode callback provided. Please provide your own in AccountMultisig constructor options. It has a parameter method where method.kind is "email" or "phone".');
+    }
+
+    async promptAndVerify() {
+        const method = await this.get2faMethod();
+        const securityCode = await this.getCode(method)
+        try {
+            const { success, res: result } = await this.verifyCode(securityCode);
+            console.log(success, result)
+            if (!success || result === false) {
+                throw new Error('Request failed with error: ' + JSON.stringify(result));
+            }
+            return typeof result === 'string' && result.length === 0 ? 'true' : result;
```

Comment:
Why are we returning `'true'` string here?

Generally it's very hard to understand what is going on with the types here. Why is there separate `success` and `result`? Why result can be `false` or string?

Reply:
result is the result from the chain, success was if contract helper could verify the code